src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -427,6 +427,25 @@
     <RemoveDir Directories="$(OutputPath)/15.0" />
   </Target>
 
+  <!-- Patch version of Newtonsoft.Json in testhost.deps.json, because it is picked up by 
+  compliance checking scripts as a false-positive vulnerability. -->
+  <Target Name="FixNewtonsoftInTesthostDepsJson"
```

Comment:
We already do some runtimeconfig rewriting in installer. I wonder if we should have this change there rather than the SDK repo: https://github.com/dotnet/installer/blob/0c1a01734963b20ba59a9da05fce0559e7286994/src/redist/targets/GenerateLayout.targets#L478

CC @dsplaisted @joeloff for their opinion. For context, the test host has a hardcoded 13.0.0 version for newtonsoft in their deps.json and that's getting flagged by a docker scanning tool. We're trying to clean up those that we can.

Reply:
In that case, I’m posting the pr in the the wrong place. I remembered something like this was done, but I didn’t remember it is in installer. 

We have Friday and Monday off here. I will post a new PR on Tuesday.